hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/HttpServer2.java

```diff
@@ -591,11 +597,11 @@ private HttpServer2(final Builder b) throws IOException {
 
     this.findPort = b.findPort;
     this.portRanges = b.portRanges;
-    initializeWebServer(b.name, b.hostName, b.conf, b.pathSpecs,
+    initializeWebServer(b, b.name, b.hostName, b.conf, b.pathSpecs,
         b.authFilterConfigurationPrefix, b.securityEnabled);
   }
 
-  private void initializeWebServer(String name, String hostName,
+  private void initializeWebServer(Builder b, String name, String hostName,
```

Comment:
Shouldn't we just provide the new boolean value skipDefaultApps from the builder or alternatively can we just supply the builder and use that in the method? I am fine with both approach, I just want to avoid using a mixture of both.